### ISSUE TITLE
panelCtrl is undefined when trying to edit a panel that uses OpenNMS data source

### ISSUE THREAD

**Issue by robison** [OpenNMS/grafana-opennms-datasource#2]
I'm seeing an issue with the plugin when trying to edit a panel that uses OpenNMS as a datasource. Grafana version is 3.0.0-beta71462173753, plugin is installed via `grafana-cli`. Error is reproducible in up-to-date versions of Chrome and Firefox.

Chrome error:
```
boot.8401a6a2.js:35 TypeError: Cannot read property 'panelCtrl' of undefined
    at new a (boot.8401a6a2.js:12)
    at Object.g [as invoke] (boot.8401a6a2.js:34)
    at l.instance (boot.8401a6a2.js:34)
    at o (boot.8401a6a2.js:34)
    at boot.8401a6a2.js:34
    at h (boot.8401a6a2.js:35)
    at boot.8401a6a2.js:35
    at n.$eval (boot.8401a6a2.js:35)
    at n.$digest (boot.8401a6a2.js:35)
    at boot.8401a6a2.js:35(anonymous function) @ boot.8401a6a2.js:35(anonymous function) @ boot.8401a6a2.js:34h @ boot.8401a6a2.js:35(anonymous function) @ boot.8401a6a2.js:35n.$eval @ boot.8401a6a2.js:35n.$digest @ boot.8401a6a2.js:35(anonymous function) @ boot.8401a6a2.js:35e @ boot.8401a6a2.js:34(anonymous function) @ boot.8401a6a2.js:34
boot.8401a6a2.js:13 could not find field in annotation:  desc
```

Firefox error:
```
Error: this.queryCtrl is undefined
a@https://grafana/public/app/boot.8401a6a2.js:12:29585
g@https://grafana/public/app/boot.8401a6a2.js:34:3587
tb/this.$get</</q<@https://grafana/public/app/boot.8401a6a2.js:34:28815
o@https://grafana/public/app/boot.8401a6a2.js:34:16119
da/<@https://grafana/public/app/boot.8401a6a2.js:34:21903
h@https://grafana/public/app/boot.8401a6a2.js:35:18030
i/<@https://grafana/public/app/boot.8401a6a2.js:35:18202
vc/this.$get</n.prototype.$eval@https://grafana/public/app/boot.8401a6a2.js:35:25715
vc/this.$get</n.prototype.$digest@https://grafana/public/app/boot.8401a6a2.js:35:24219
vc/this.$get</n.prototype.$apply@https://grafana/public/app/boot.8401a6a2.js:35:26000
f@https://grafana/public/app/boot.8401a6a2.js:35:845
q@https://grafana/public/app/boot.8401a6a2.js:35:2695
Ib/</v.onload@https://grafana/public/app/boot.8401a6a2.js:35:3238
```

**Comment by j-white** [comment]
Thanks for the report, it looks like they have changed some of their APIs.

I can confirm that the plugin works with grafana-3.0.0-beta51460725904, you can downgrade to that version until I get a new release ready.

**Comment by j-white** [comment]
This is fixed in v2.0.1. It will be release once https://github.com/grafana/grafana-plugin-repository/pull/17 gets merged.

**Comment by j-white** [comment]
By the way, that PR was merged, reinstalling the plugin with `sudo grafana-cli plugins install opennms-datasource` should solve the issue now.
